joomla/joomla-cms
# Use setDocumentTitle method of JViewLegacy to set document title

Pull Request for Issue # .

### Summary of Changes
Since Joomla 3.6 , JViewLegacy has implemented the method setDocumentTitle https://github.com/joomla/joomla-cms/blob/staging/libraries/legacy/view/legacy.php#L854-L873 which is used to set document title base on site configuration. However, in many views in Joomla code base, we still use manual code, repeating code to set page title. This PR solves it

### Testing Instructions
A real test would be check all views changes by this PR and make sure the document title is not changed before and after patch. However, honestly, I don't think we can expect that kind of test here because it would take much time

So I hope developers can help reviewing the code of the PR and confirm that it is expected changes. If you review code, please help reviewing every file carefully.

### Documentation Changes Required
None
